programs/ziti-edge-tunnel/windows-service.c

```diff
@@ -456,7 +456,7 @@ DWORD LphandlerFunctionEx(
             ReportSvcStatus(SERVICE_STOP_PENDING, NO_ERROR, 0);
 
             // stops the running tunnel service
-            ziti_tunnel_async_send(NULL, scm_service_stop, NULL);
+            scm_service_stop();
```

Comment:
This function is exposed in the windows-service.h file. If we use the scm_service_stop_event function which accepts uv_loop_t here, we will have to import additional uv libraries in the windows file.